content/lua/concepts/tables/terms/getn/getn.md

```diff
@@ -0,0 +1,83 @@
+---
+Title: 'getn()'
+Description: 'Returns the number of elements in the table passed.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Tables'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
```

Comment:
Please update the spacing in the entry. There should never be multiple empty lines between content, and there should always be one empty line after a header or code block